resources/queries/targetedms/QCGroupingPrecursorMolecules.sql

```diff
@@ -0,0 +1,40 @@
+-- This query for Molecules displays under QC Summary menu 'Include or Exclude Peptides/Molecules'
+
+SELECT
+    *,
+    (CASE
+         WHEN molPrec.precursorIdentifier IN (SELECT precursorIdentifier FROM targetedms.ExcludedPrecursors) THEN 'Excluded'
+         ELSE 'Included'
+        END) AS markedAs,
+FROM
+     (SELECT
+        ((CASE WHEN molPrec.MoleculeId.customIonName IS NOT NULL THEN molPrec.MoleculeId.customIonName
```

Comment:
Unfortunately, you can have any combination of null and non-null values for these two columns. We should include both if they're non-null

Reply:
done